src/utils/webSockets.observable.js

```diff
@@ -1,4 +1,4 @@
-import { Observable } from 'rxjs/Observable';
```

Comment:
You have been changing this import all around the file, but there doesn't seem to exist any change on the rxjs version... was it wrong all the time?